spec.html

```diff
@@ -7875,6 +7880,251 @@ <h1>Forward Progress</h1>
       <p>This, along with the liveness guarantee in the memory model, ensures that all ~SeqCst~ writes eventually become observable to all agents.</p>
     </emu-note>
   </emu-clause>
+
+  <emu-clause id="sec-weakref-processing-model">
+    <h1>Processing model of WeakRef and FinalizationRegistry objects</h1>
+    <emu-clause id="sec-weakref-invariants">
+      <h1>Objectives</h1>
+
+      <p>
+        This specification does not make any guarantees that any object will be garbage collected.
+        Objects which are unreachable from ECMAScript may be released after long periods
```

Comment:
I'm not sure if it will be quite that simple, since we define liveness for _sets_ of objects. A given object may be a member of both a live set and a non-live set.